pkg/storage/unified/sql/backend.go

```diff
@@ -214,28 +215,46 @@ func (b *backend) initPruner(ctx context.Context) error {
 		MaxWait:    time.Minute * 5,
```

Comment:
🤔  the MinWait should maybe be longer than 30s... or maybe we add a clause that the rv must be 5 min old also?

If we are too aggressive, we would break the k8s compatibility.  So this should likely match the etcd bookmark time